### ISSUE TITLE
Show-ParameterSetInfo: columns are misaligned on Unix-like platforms

### ISSUE THREAD

**Issue by mklement0** [EliziumNet/Loopz#146]
For reference, here's correct output from Windows Terminal:

![image](https://user-images.githubusercontent.com/588825/115035920-0a9b4700-9e9b-11eb-8cbd-3c42e37e345c.png)

On both macOS 10.15.7 and Ubuntu 18.04, the columns are misaligned:

* macOS

![image](https://user-images.githubusercontent.com/588825/115036113-36b6c800-9e9b-11eb-8200-757811a5ac43.png)

* Ubuntu

![image](https://user-images.githubusercontent.com/588825/115036175-446c4d80-9e9b-11eb-82c0-6b5f87f839f3.png)

**Comment by plastikfan** [comment]
Yeah, I know about this and this is an outstanding issue that is out of my control. This is caused by another issue(s) in xterm.js/xterm.js [Issue 3304](https://github.com/xtermjs/xterm.js/issues/3304 and [Issue 2568](https://github.com/xtermjs/xterm.js/pull/2568)) which affects macOS/Linux and even vscode on windows. This misalignment occurs in vscode too, even on windows.

At the moment, I have a command *Test-WinHostSupportsEmojis*, which defaults to enabling emojis on Linux and macOS. I should probably default to not using emjois for Linux and macOS, until these types of issues have been fixed in xterm.js. So that is what I'll implement for this issue.

**Comment by christianparpart** [comment]
Hi. I just stumbped across your ticked sure to my personal interest in grapheme Cluster support in terminals and a small chat with @jerch. Personally i think WT is the only one doing it right here. If a TE renders an emoji symbol that one is too be treated as (East Asian) Wide character (i.e. double width = 2 cells).

While ah TEs above den to render correctly they do NOT position the cursor correctly. The increment the current column only by one instead of 2, therefore your app looks broken. 

I think there is a future proof way to convince TE devs to do it like WT does (i think i will propose that in jerch's referenced ticked).

You can work around that btw by using tabstops instead of space characters to move your cursor forwards, so your app looks the same also in non supporting TEs.

If you need tabstops not equal to modulo 8, you can explicitly set tabstops upfront (TBC, HTS).

ref: https://vt100.net/docs/vt510-rm/HTS.html

**Comment by plastikfan** [comment]
thanks for your contribution @christianparpart. Looking at that vt100 page though doesnt mean anything to me yet. I have found this article on msdn [Virtual Terminal Sequences](https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences), which hopefully enable me to make sense of how to apply the HTS as you recommend. Cheers.
